packages/web-pkg/src/composables/download/useDownloadFile.ts

```diff
@@ -29,30 +27,26 @@ export const useDownloadFile = () => {
       } else {
         url = client.fileVersions.getFileVersionUrl(file.fileId, version)
       }
-      const accessToken = store.getters['runtime/auth/accessToken']
-      headers = { Authorization: 'Bearer ' + accessToken }
     }
 
     // download with signing enabled
     if (isUserContext && unref(isUrlSigningEnabled)) {
+      const httpClient = clientService.httpAuthenticated
       try {
-        const response = await fetch(url, {
-          method: 'HEAD',
-          headers
-        })
+        const response = await httpClient.head(url)
         if (response.status === 200) {
           const signedUrl = await client.signUrl(url)
           triggerDownloadWithFilename(signedUrl, file.name)
           return
         }
       } catch (e) {
         console.error(e)
+        store.dispatch('showErrorMessage', {
+          title: $gettext('Download failed'),
+          desc: $gettext('File could not be located'),
+          error: e
```

Comment:
shouldn't it be `e.error`? like in the first change?

Reply:
what you do you mean with, like in the first change? why should the error object contain an error field?

Reply:
my bad, its fine